### PR TITLE
Compile assets at builtime

### PR DESCRIPTION
**Pull request by armandfardeau**
Hi @MoretS 

I removed the assets that had been generated and moved them to the docker builtime. 

In the previous configuration they would have been out-of-date all the time which is not good and would have caused a lot of problems.

I looked in your dockerfile and you don't set the RAILS_ENV variable, why? It's a waste to create images in development, isn't it?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,6 +23,10 @@ WORKDIR /app
 
 RUN bundle install
 
+# Perform assets compilation
+RUN yarn install
+RUN SECRET_KEY_BASE=dummy bundle exec rails assets:precompile
```
Comment (Quentinchampenois) [inline Dockerfile:28]: Hello,

Are we sure about a dummy `secret_key_base` in the Dockerfile ?

**Review by Quentinchampenois — COMMENTED** [review]
Hi, 

@MoretS Could you take a look too to this PR please ? :)

**Comment by Quentinchampenois** [comment]
@armandfardeau Should we merge ?

**Comment by paulinebessoles** [comment]
@armandfardeau  j'ai changé la base branch pour mettre la rc 0.25 car la develop va changer pour récup la master

**Pull request merged**